mcode/mcode-coverage-checker
# STEAM-991: Adding component to switch data source

# Description
Issue: STEAM-991

This PR adds a new dropdown select component that allows the user to switch between files uploaded on the file upload page.

## Important Changes
- `App.js` now pulls the list of files from `recoil_state.js` and computes coverage checking based on the currently selected file from that list
- Moved data from `fullbundle.json` into `bundle1.json` of the recoil state for testing purposes
- New component, `FileSelect.js`, created to allow the user to pick between files
- `Rankings.js` refactored to actually update on changes to `coverageData`
- Temporary Links added to the main and file upload pages to allow going back and forth between the two

## Testing Recommendations
- Switch between files and see that the main visualization and rankings section properly update
- Navigate to the file upload page and see that adding and removing files is properly reflected in the file select component on the main page

# Checklists

**Submitter:**

@ACCT1 :
